src/useSubscription/useSubscriptionDevtools.ts

```diff
@@ -0,0 +1,243 @@
+import type {
+  DevtoolsPluginApi,
+  CustomInspectorNode,
+  CustomInspectorState,
+  TimelineEvent,
+  PluginSettingsItem,
+  ExtractSettingsTypes} from '@vue/devtools-api'
+import Channel from './models/channel'
+import { ComponentInternalInstance, getCurrentInstance, nextTick } from 'vue'
+
+function throttle(fn: Function, wait: number) {
+  let isThrottled = false
+  let invokedDuringThrottle = false
+
+  function wrapper(this: unknown) {
+    const context = this
+    if (isThrottled) {
+      invokedDuringThrottle = true
+      return;
+    }
+    isThrottled = true;
+
+    fn.apply(context, arguments)
+
+    setTimeout(function() {
+      isThrottled = false; // (3)
+      if (invokedDuringThrottle) {
+        wrapper.apply(context);
+        invokedDuringThrottle = false
+      }
+    }, wait);
+  }
+
+  return wrapper;
+}
+
+const SUBSCRIPTIONS_INSPECTOR_ID = 'prefect-vue-compositions-subscriptions'
+const SUBSCRIPTIONS_TIMELINE_LAYER_ID = 'prefect-vue-compositions-subscriptions'
+
+export const SUBSCRIPTION_DEVTOOLS_SETTINGS = {
+  // timelineColor: {
+  //   label: 'Timeline Color',
+  //   type: 'choice',
+  //   defaultValue: 0xFF69B4,
+  //   options: [
+  //     { value: 0x4fc08d, label: 'Vue Green' },
+  //     { value: 0x175cd3, label: 'Prefect Blue' },
+  //     { value: 0xFF69B4, label: 'Hot' }
+  //   ],
+  //   component: 'button-group'
+  // }
```

Comment:
We can add plugin settings like this https://devtools.vuejs.org/plugin/plugins-guide.html#plugin-settings

Here I was going to add a toggle to switch the color on the timeline but... changing the layer's color after having created it already isn't supported... 😢 . I thought i'd leave this scaffolding here tho incase some other setting comes to mind to add in. 

1 thought I had was a giant list of all timeline events so you could filter down which ones get emitted/tracked.